src/components/ContentUploader/ContentUploader.js

```diff
@@ -480,41 +652,160 @@ class ContentUploader extends Component<DefaultProps, Props, State> {
+     * 
+     * @return {void}
+     */
+    resetUploadsManagerItemsWhenUploadsComplete = (): void => {
+        const { view, items, isUploadsManagerExpanded } = this.state;
+        const { windowView, onUploadsManagerCancel } = this.props;
+
+        // Do not reset items when upload manger is expanded or there're uploads in progress
+        if ((isUploadsManagerExpanded && windowView && !!items.length) || view === VIEW_UPLOAD_IN_PROGRESS) {
+            return;
+        }
+
+        items.forEach((item) => {
+            onUploadsManagerCancel(item);
+        });
+
+        this.setState({
+            items: []
+        });
+    };
+
+    /**
+     * Adds file to the upload queue and starts upload immediately
+     * 
+     * @param {Array<UploadFileWithAPIOptions | File>} files - Files to be added to upload queue
+     * @return {void}
+     */
+    addFilesWithOptionsToUploadQueueAndStartUpload = (files: Array<UploadFileWithAPIOptions | File>): void => {
+        this.addFilesToUploadQueue(files, true, this.upload);
+    };
+
     /**
      * Renders the content uploader
      *
      * @inheritdoc
      * @return {Component}
      */
     render() {
-        const { language, messages, onClose, className, measureRef, isTouch, fileLimit }: Props = this.props;
-        const { view, items, errorCode }: State = this.state;
+        const {
+            language,
+            messages,
+            onClose,
+            className,
+            measureRef,
+            isTouch,
+            fileLimit,
+            windowView
+        }: Props = this.props;
+        const { view, items, errorCode, isUploadsManagerExpanded }: State = this.state;
 
         const hasFiles = items.length !== 0;
         const isLoading = items.some((item) => item.status === STATUS_IN_PROGRESS);
-        const styleClassName = classNames('buik buik-app-element bcu', className);
+        const styleClassName = classNames('bcu', className, {
+            'buik-app-element': !windowView,
+            buik: !windowView
+        });
 
         return (
             <Internationalize language={language} messages={messages}>
-                <div className={styleClassName} id={this.id} ref={measureRef}>
-                    <DroppableContent
-                        addFiles={this.addFilesToUploadQueue}
-                        allowedTypes={['Files']}
-                        items={items}
-                        isTouch={isTouch}
-                        view={view}
-                        onClick={this.onClick}
-                    />
-                    <Footer
-                        hasFiles={hasFiles}
-                        isLoading={isLoading}
-                        errorCode={errorCode}
-                        fileLimit={fileLimit}
-                        onCancel={this.cancel}
-                        onClose={onClose}
-                        onUpload={this.upload}
-                    />
-                </div>
+                {windowView
```

Comment:
Since `windowView` aka WebApp is pretty tied to the UploadsManager, how about we just rename this `useUploadsManager` and let that be the distinction?

Reply:
yeah I agree, also considering that webapp probably will be the only user of UploadsManager for a while